### ISSUE TITLE
Call discoverNodeNeighbors for the entire network

### ISSUE THREAD

**Issue by duylong** [zwave-js/zwave-js-ui#3549]
Hi,

Is there a way to call a discoverNodeNeighbors for the entire network? In the same way as the "Rebuild routes" option with the possibility of doing it only for all powered modules.

Best regards,

**Comment by robertsLando** [comment]
You can do that by using a custom driver function but that would require you to manually wake up battery powered devices also there may be some connectivity issues during this... why would you do that?

cc @AlCalzone

**Comment by AlCalzone** [comment]
It's already done as part of rebuilding routes.

**Comment by duylong** [comment]
I wanted to diagnose my network in order to improve it and reduce the number of hops as much as possible.

In this task, I saw that we could manually force the discovery of neighbors in "Network Graph". I've already rebuilt the roads, I didn't want to do it again, it's time-consuming and error-prone. It seemed like a bad idea to me. 

The discovery of neighbors seems faster, so I am looking for the possibility of doing it in batches before looking for inconsistencies node by node.
